contracts/upgradeable_contracts/erc20_to_erc20/ForeignBridgeErcToErc.sol

```diff
@@ -18,10 +21,31 @@ contract ForeignBridgeErcToErc is BasicForeignBridgeErcToErc, ERC20Bridge {
             _erc20token,
             _requiredBlockConfirmations,
             _gasPrice,
-            _maxPerTxHomeDailyLimitHomeMaxPerTxArray,
+            _dailyLimitMaxPerTxMinPerTxArray,
+            _homeDailyLimitHomeMaxPerTxArray,
             _owner,
             _decimalShift
         );
         return isInitialized();
     }
+
+    function _relayTokens(address _sender, address _receiver, uint256 _amount) internal {
```

Comment:
is this code common for `erc20-to-native` and `erc20-to-erc20` bridge modes? Bearing in mind [this comment](https://github.com/poanetwork/poa-bridge-contracts/pull/302#discussion_r337174872) could it be even more common?

Reply:
Right, I'll move the methods to a common module